src/services/addressResolver/index.ts

```diff
@@ -49,14 +46,19 @@ const get = async ({
 };
 
 export const entityLookup = async ({
+  limit,
+  offset,
   query,
   endpoint,
   apiHeader,
   apiKey,
-}: EntityLookupProps): Promise<AddressBookThirdPartyResultsProps[]> => {
-  const url = `${endpoint}search?q=${query}`;
+}: EntityLookupProps): Promise<EntityLookupResponseProps> => {
+  const anyLimit = limit !== undefined ? `&limit=${limit}` : "";
+  const anyOffset = offset !== undefined ? `&offset=${offset}` : "";
+
+  const url = `${endpoint}search?q=${query}${anyOffset}${anyLimit}`;
   const response = await get({ url, apiHeader, apiKey });
-  return response.data.identities;
+  return response.data;
```

Comment:
want to use query-string library? then u can do:

```
  const url = queryString.stringifyUrl({
    url: `${endpoint}search`,
    query: {
      q: query,
      limit,
      offset,
    },
  });
```

Reply:
ok